gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -33,32 +33,33 @@
 
 from __future__ import absolute_import
 
-import argparse
 import datetime
-import enum
 import logging
 import sys
 import tempfile
+from argparse import Namespace  # pylint: disable=unused-import
+from typing import List  # pylint: disable=unused-import
 
 import apache_beam as beam
+from apache_beam import Pipeline  # pylint: disable=unused-import
 from apache_beam.io.filesystems import FileSystems
 from apache_beam.options.pipeline_options import GoogleCloudOptions
 from apache_beam.options.pipeline_options import PipelineOptions
 from apache_beam.options.pipeline_options import StandardOptions
+from apache_beam.pvalue import PCollection  # pylint: disable=unused-import
 
-from gcp_variant_transforms.beam_io import vcf_header_io
-from gcp_variant_transforms.beam_io import vcfio
+from gcp_variant_transforms import general_process
 from gcp_variant_transforms.libs import metrics_util
 from gcp_variant_transforms.libs import vcf_header_parser
 from gcp_variant_transforms.libs.variant_merge import merge_with_non_variants_strategy
 from gcp_variant_transforms.libs.variant_merge import move_to_calls_strategy
 from gcp_variant_transforms.libs import processed_variant
 from gcp_variant_transforms.options import variant_transform_options
+from gcp_variant_transforms.options.variant_transform_options import MergeOptions
```

Comment:
The styleguide actually recommends not importing classes and just importing modules...we've not been using that in tests and the unsuced imports though... but, this is an interesting case; could you use the full path in `type`? Given that it's only once place, then it shouldn't be that bad i guess.